crates/oxc_semantic/Cargo.toml

```diff
@@ -30,6 +30,7 @@ oxc_syntax = { workspace = true }
 assert-unchecked = { workspace = true }
 indexmap = { workspace = true }
 itertools = { workspace = true }
+more-asserts = { workspace = true }
```

Comment:
FYI this is slightly against our policy of minimal third-party crates. https://oxc.rs/docs/contribute/rules.html#development-policy